spring-kafka-test/src/main/java/org/springframework/kafka/test/rule/KafkaEmbedded.java

```diff
@@ -278,7 +278,7 @@ public void addTopics(String... topics) {
 	 * Create an {@link AdminClient} invoke the callback and reliable close the
```

Comment:
Please, add a comma before `invoke` and we merge your PR.

Thanks

Reply:
Sure ok will do